daal4py/sklearn/model_selection/split.py

```diff
@@ -0,0 +1,201 @@
+from sklearn.utils import indexable, _safe_indexing

```

Comment:
Did split.py from model_selection have different implementation for different version sklearn (0.22, 0.23, etc) ?

Reply:
I think you need to make similar to other algorithms

Reply:
Split has stable API and implementation in last versions.